CleverFerret/src/main/java/com/universalmedialibrary/ui/settings/SettingsSubPages.kt

```
@@ -0,0 +1,1119 @@
+
+private suspend fun clearCache(context: android.content.Context) {
+    runCatching { context.cacheDir.deleteRecursively() }
+    runCatching { context.externalCacheDir?.deleteRecursively() }
+}
+
+private fun Long.toHumanReadable(): String {
+    val kb = 1024.0
+    val mb = kb * 1024
+    val gb = mb * 1024
+    return when {
+        this >= gb -> String.format("%.2f GB", this / gb)
```

detekt (1.23.8)

[warning] String.format("%.2f GB", this / gb) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)